### PR TITLE
histogram: brighten primaries

### PR DESCRIPTION
**Pull request by dtorop**
These colors are closer in saturation to the appearance prior to #6135. This is as suggested by @Nilvus, see https://github.com/darktable-org/darktable/pull/6135#issuecomment-727248993.

Before #6135:

image

With this PR:

image

After #6135:

image

### CODE REVIEW

**Comment by TurboGit** [comment]
Looks ok to me! I'll let @Nilvus comment before merging.

**Comment by Nilvus** [comment]
I will test that this day but to see screenshots, that seems much better

**Comment by chhil** [comment]
Left with the change on linux with mouse over on shadows area. Right without the color change on windows.
image

**Comment by Nilvus** [comment]
@dtorop: I have something strange with your branch here. I can't go to darkroom. darktable crashes after few seconds freeze. See backtrace I have:

[darktable_bt_5W9AU0.txt](https://github.com/darktable-org/darktable/files/5542375/darktable_bt_5W9AU0.txt)

I will try your changes here by copy/paste them on actual master that works great.

**Comment by Nilvus** [comment]
@dtorop: just forget my last comment. The issue is in master, not your branch.

Just tested so your new colors with a working master and it's just much much better and now good for me. Thanks for improving that. @TurboGit: you could merge that.

**Comment by TurboGit** [comment]
Should be fixed by my commit 325b696289443de9bbdc9121a668dc4f1d76f9c0.

**Comment by TurboGit** [comment]
Ok, this is fine by me. @Nilvus : any objection to merge ?

**Review by TurboGit — APPROVED** [review]
Thanks !

**Comment by dtorop** [comment]
Hope these colors are good for now, and 3.4. There can be more thought later.

Looking at the pre- PR 6135 colors, they look over-saturated (magenta to red transition, for example, is hard to see). For this PR I used colors which are still pretty saturated. They should be fine on a display with approx. Adobe RGB gamut. For a display with smaller gamut (e.g. sRGB), they may be out of gamut.

My guess is that a decade ago, most display had sRGB-ish gamuts, and the histogram colors were totally reasonable. As displays have gotten wider gamut, we've slowly gotten used to a brighter and brighter histogram...

**Comment by Nilvus** [comment]
> Hope these colors are good for now, and 3.4. There can be more thought later.

Yes, they are really good now.

> Looking at the pre- PR 6135 colors, they look over-saturated (magenta to red transition, for example, is hard to see). For this PR I used colors which are still pretty saturated. They should be fine on a display with approx. Adobe RGB gamut. For a display with smaller gamut (e.g. sRGB), they may be out of gamut.
> 
> My guess is that a decade ago, most display had sRGB-ish gamuts, and the histogram colors were totally reasonable. As displays have gotten wider gamut, we've slowly gotten used to a brighter and brighter histogram...

Many screens doesn't even have a real 100% sRGB gamut and 100% Adobe RGB gamut remains quite expensive screens. I have a great screen in a reasonable price for photo which is the Asus ProArt PA24AC. A 100% sRGB gamut one and, as I remember, around 74% of Adobe RGB gamut. And colors are now great on histogram. Indeed previous were over saturated and new ones are great with that PR.

**Comment by dtorop** [comment]
> And colors are now great on histogram.

This is great to hear. I think I still may have to change things because of #6945, but should be able to keep the existing colors, or something close. I also feel good about how they look.

**Pull request merged**
